x-pack/plugins/reporting/server/browsers/download/checksum.ts

```diff
@@ -7,8 +7,13 @@
 
 import { createHash } from 'crypto';
 import { createReadStream } from 'fs';
+import { Readable } from 'stream';
 
-import { readableEnd } from './util';
```

Comment:
`util` was removed because it didn't contain anything significant and the code inside is easy enough to define in-line where it is needed.